packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol

```diff
@@ -141,6 +141,7 @@ contract OptimisticAssertor is Lockable, OptimisticAssertorInterface, Ownable {
     }
 
     function disputeAssertionFor(bytes32 assertionId, address disputer) public {
+        address _disputer = disputer == address(0) ? msg.sender : disputer;
```

Comment:
I wonder what would be the usecase of defaulting to msg.sender instead of reverting `disputer == address(0)`

Reply:
we might want to create wrapped function `disputeAssertionFor(bytes32 assertionId)` that passes zero disputer address to `disputeAssertionFor` similarly as we do in `assertTruth` with default parameters.